localhost

```diff
@@ -1 +1 @@
-localhost
+localhost ansible_python_interpreter=/usr/local/bin/python2.7
```

Comment:
Do you have any logs when it did not work?  I did not have problems running ansible with the current contents of `group_vars/all`...